### ISSUE TITLE
Error getting v2 registry: Get https://registry-1.docker.io/v2/: Bad Request

### ISSUE THREAD

**Issue by kishorsn77** [docker/for-win#844]
### Expected behavior
When i run >docker run hello-world, the image has to be downloaded and a container kicks in.
### Actual behavior
When i run >docker run hello-world, The below error message is seen -

WARNING: Error loading config file:C:\Users\Kishor_SN\.docker\config.json - EOF
Unable to find image 'hello-world:latest' locally
docker: Error response from daemon: Get https://registry-1.docker.io/v2/: Bad Request.
See 'docker run --help'.
### Information

It looks like the client is unable to authenticate on public registry (https://registry-1.docker.io/v2/)

  - Diagnostic ID from "Diagnose & Feedback" in the menu.
0DF3BA8C-AC0A-49F7-951F-A553128C046F/2017-06-23_15-36-48
  - a reproducible case if this is a bug, Dockerfiles FTW
  - page URL if this is a docs issue or the name of a man page
  - host distribution and version (Windows version, build number, etc)
Microsoft Windows 10 Build 14393.1198

### Steps to reproduce the behavior

  1. Run docker command -> docker run hello-world
  2. Get the below error message - 
WARNING: Error loading config file:C:\Users\Kishor_SN\.docker\config.json - EOF
Unable to find image 'hello-world:latest' locally
docker: Error response from daemon: Get https://registry-1.docker.io/v2/: Bad Request.
See 'docker run --help'.

**Comment by jasonbivins** [comment]
Hi @kishorsn77 
Are you able to use the docker login command to reach the registry?
Have you made any changes to the config.jason file through the GUI?

**Comment by kishorsn77** [comment]
Hi Jason,
I tried using docker login, but was getting the same error. And i did not
change anything in config.jason file.

It wasn't working under two conditions -

1. When my system is in corporate network. Even though i entered the proxy
details, i get the "Bad Request" message when pulling the images. Need some
solution for this.
2. When my system is out of corporate network. This option wasn't working
until today. When i tried this a while ago, it started working. But again,
hit another issue. THe antivirus isn't allowing to install the container.
Any workaround for this?

**Comment by jasonbivins** [comment]
Hi @kishorsn77   
Try creating an exception in your antivirus for the Docker service.

Can you have your network admins check to see if you are getting blocked by the corporate proxy or firewall ?
